docs/api.rst

```diff
@@ -341,99 +440,99 @@ Validators can be globally disabled if you want to run them only in development
 Validators
 ----------
 
-``attrs`` comes with some common validators in the ``attrs.validators`` module:
+``attrs`` comes with some common validators in the ``attrs.validators`` module.
+All objects from ``attrs.converters`` are also available from ``attr.converters``.
```

Comment:
@hynek Hi! I just wanted to let you know that this line about converters does not probably belong here (it looks like a duplicate of a line down below). Thanks for attrs and for its wonderful docs!

Reply:
thanks, fixed in e188af7bc4df99eb84772b0ad0f80c4e11f740b4!